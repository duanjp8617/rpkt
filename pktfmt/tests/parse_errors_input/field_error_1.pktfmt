packet FakeProto {
    header = [
        fake_field1 = Field {
            bit = 65, 
        },
    ]
}

