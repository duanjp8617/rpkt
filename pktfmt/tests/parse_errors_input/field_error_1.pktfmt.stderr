ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/field_error_1.pktfmt 3:23-5:9
                          ~~~~~~~
3 |         fake_field1 = Field {
4 |             bit = 65, 
5 |         },
    ^^^^^^^^^
field error 1:
invalid bit 65, the following bit values are invalid:
1. bit == 0,
2. bit > 64 && bit % 8 != 0, 
3. bit > 8388600
